Tools/machines/perlmutter-nersc/Containerfile

```diff
@@ -0,0 +1,151 @@
+# Base System and Essential Tools Installation
+FROM nvidia/cuda:12.6.0-devel-ubuntu22.04 AS base
```

Comment:
To test: we could use `-devel` to build all dependencies but then copy their artifacts in the `-runtime` image. To check if those are much smaller or not.